frontend/src/scenes/urls.ts

```diff
@@ -76,21 +75,28 @@ export const urls = {
         filters?: AnyPartialFilterType,
         dashboardId?: DashboardType['id'] | null,
         query?: string | Record<string, any>
-    ): string =>
-        combineUrl('/insights/new', dashboardId ? { dashboard: dashboardId } : {}, {
+    ): string => {
+        if (query) {
+            return combineUrl(`/sql`, {}, query ? { q: typeof query === 'string' ? query : JSON.stringify(query) } : {})
+                .url
+        }
```

Comment:
This isn't right, as insights use `query` too. You can see this in Web analytics, where in this PR "Open as new insight" opens trend insights in… SQL studio. Only queries with `kind: "HogQLQuery"` should be opened (or generally _openable_) in the SQL scene.